templates/lb-fargate-service/params.json.tmpl

```diff
@@ -13,10 +13,8 @@
     "LogRetention": "30",
     "AddonsTemplateURL": "",
     "HealthCheckPath": "{{.App.HealthCheckPath}}"
-  },
-  "Tags": {
-    "ecs-project": "{{.Env.Project}}",
-    "ecs-environment": "{{.Env.Name}}",
-    "ecs-application": "{{.App.Name}}"
-  }
+  },{{$n := len .Tags}}{{if gt $n 0}}
+  "Tags": { {{range $i, $t := .Tags}}
+    {{if eq (inc $i) $n}}"{{$t.Key}}": "{{$t.Value}}"{{else}}"{{$t.Key}}": "{{$t.Value}}",{{end}}{{end}}
```

Comment:
lol I thought it should be fine to have the last one with "," but apparently it is not. Can we do {{if eq ($i+1) $n}} or at the beginning {{$n := len .Tags + 1}}, so that we don't need the "inc"?

Reply:
You can't do arithmetic in go templates >_> that's why I had to write that weird "inc" function

Reply:
> You can't do arithmetic in go templates >_>

That is so annoying :)

Reply:
Go is perfect and has no flaws.